hvac/api/secrets_engines/transit.py

```diff
@@ -757,6 +757,7 @@ def sign_data(
         hash_input,
         key_version=None,
         hash_algorithm=None,
+        batch_input=None,
```

Comment:
Please move this to the end of the argument list, otherwise it will break folks using positional arguments.